pkg/vpcmodel/drawioOutput.go

```diff
@@ -119,11 +141,19 @@ func (d *DrawioOutputFormatter) WriteOutput(c1, c2 *VPCConfig,
 	var err error
 	switch uc {
 	case AllEndpoints:
-		d.init(c1, conn)
-		d.createDrawioTree()
-		err = drawio.CreateDrawioConnectivityMapFile(d.gen.Network(), outFile)
-	case AllSubnets, SingleSubnet:
-		err = errors.New("SubnetLevel/SingleSubnet use case not supported for draw.io format")
+		var gConn *GroupConnLines
+		if conn != nil {
+			gConn = conn.GroupedConnectivity
+		}
+		return d.writeOutputGeneric(c1, gConn, outFile, false)
+	case AllSubnets:
+		var gConn *GroupConnLines
+		if subnetsConn != nil {
```

Comment:
Why is this check, of comparing to nil, required here? It is not performed in the other formats.